app/models/roster_entry.rb

```diff
@@ -6,4 +6,32 @@ class RosterEntry < ApplicationRecord
 
   validates :identifier, presence: true
   validates :roster,     presence: true
+
+  def self.to_csv
+    header = %i[identifier github_username name]
+    roster_array = []
+
+    CSV.generate(headers: true, col_sep: ",", force_quotes: true) do |csv|
+      csv << header
+
+      all.sort_by(&:identifier).each do |entry|
+        row   = []
+        login = ""
+        name  = ""
+        if entry.user
+          login = entry.user.github_user.login
+          name  = entry.user.github_user.name ? entry.user.github_user.name : ""
+        end
+
+        row << entry.identifier
+        row << login
+        row << name
+
+        roster_array << row
```

Comment:
could you do without `row` and just have `roster_array << [entry.identifier, login, name]`?